py3-pip/upgrade-requests-to-2.32.4.patch

```diff
@@ -0,0 +1,215 @@
+From 036b1e0f4271ebb6068a75fd0869d7eecd48162c Mon Sep 17 00:00:00 2001
```

Comment:
Would it be possible to include the URL for this patch in the header of this file?